source/tool/chipsec/helper/linux/helper.py

```diff
@@ -99,16 +104,46 @@ def __del__(self):
 # Driver/service management functions
 ###############################################################################################
 
+    # This function load CHIPSEC driver. (implement functionality from run.sh)
+    def load_chipsec_module(self):
+        page_is_ram = ""
+        if self.GET_PAGE_IS_RAM:
+            page_is_ram = self.get_page_is_ram()
+            if not page_is_ram:
+                if logger().VERBOSE:
+                    logger().log("Cannot find symbol 'page_is_ram'")
+        driver_path = os.path.join(chipsec.file.get_main_dir(), ".." , "drivers" ,"linux", "chipsec.ko" )
+        subprocess.check_output( [ "insmod", driver_path,"a1=0x%s" % page_is_ram ] )
+        uid = os.getenv('SUDO_UID')
+        gid = os.getenv('SUDO_GID')
+        if uid and gid:
+            os.chown(self.DEVICE_NAME, int(uid), int(gid))
+        os.chmod(self.DEVICE_NAME, 600)
+        if os.path.exists(self.DEVICE_NAME):
+            self.driver_loaded = True
```

Comment:
init function sets driver_loaded to True. should probably remove this line from here